server/src/main/java/org/candlepin/controller/Entitler.java

```diff
@@ -204,11 +204,14 @@ public void adjustEntitlementQuantity(Consumer consumer, Entitlement ent,
         Consumer consumer = data.getConsumer();
         Owner owner = data.getOwner();
 
-        if (!consumer.isDev() && owner.isAutobindDisabled()) {
-            log.info("Skipping auto-attach for consumer '{}'. Auto-attach is disabled for owner {}.",
-                consumer, owner.getKey());
-            throw new AutobindDisabledForOwnerException(i18n.tr("Auto-attach is disabled for owner \"{0}\".",
-                owner.getKey()));
+        if (!consumer.isDev() && owner.isAutobindDisabled() || owner.isContentAccessEnabled()) {
```

Comment:
Just for clarification, is the intent here to be that the contentAccessEnabled be a valid condition even when the isDev and isAutobindDisabled checks fail?

That is, what we have here is equivalent to:
```if ((!consumer.isDev() && owner.isAutobindDisabled()) || owner.isContentAccessEnabled()) {```

Is that what's intended?